packages/snap-preact/components/src/components/Molecules/Select/Select.tsx

```diff
@@ -16,11 +16,10 @@ import { Lang, useLang } from '../../../hooks';
 import deepmerge from 'deepmerge';
 import Color from 'color';
 
-const CSS = {
-	select: ({ color, backgroundColor, borderColor, theme }: Partial<SelectProps>) => {
-		const lightenedPrimary = new Color(backgroundColor || color || theme?.variables?.colors?.primary).lightness(95);
-
-		return css({
+const defaultStyles: StyleScript<SelectProps> = ({ color, backgroundColor, borderColor, theme }) => {
+	const lightenedPrimary = new Color(backgroundColor || color || theme?.variables?.colors?.primary).lightness(95);
+	return css({
+		'&.ss__select--nonnative': {
```

Comment:
Could use `native` prop to return different styles instead of relying on this classname.